### DISCUSSION TITLE
Undefined in dashboard/app

### DISCUSSION THREAD

**Discussion by Zarvhos** [pockethost/pockethost#245] [Q&A] [answered]
Dear PocketHost Team,

I hope this message finds you well. I wanted to inform you about a minor issue I encountered while using PocketHost. Instead of being shown japanime-dev.pockethost.io, I am currently seeing undefined.pockethost.io. I believe there might be a technical glitch causing this discrepancy.

Image : 
<img width="649" alt="image" src="https://github.com/benallfree/pockethost/assets/61838403/c2027243-11b1-4b2a-a35d-18eb43bb6a5a">

I edited the the app name instantly after creating the instance, it might be coming from it ?

PS : I want to take a moment to express my appreciation for the PocketHost project. It has been a valuable tool for quickly launching my pocketBase app.

**Comment by benallfree** [comment, marked as answer]
Thanks for the report! I have a feeling something got knocked loose in the most recent upgrade. I’ll check into it!

**Comment by benallfree** [comment]
Fixed in 0.8.2
